index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/8f721d437f.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <main class="body-wrapper">
+      <!-- mobile nav section -->
+      <header class="mobile-nav">
+        <a href="/" class="logo">Welcome</a>
```

Comment:
- [ ] [OPTIONAL] The font of the welcome logo on the Figma design is `Inter` but you're using `Poppins`.

Reply:
@topeogunleye Thank you soo much

Reply:
@topeogunleye Please brother I am done can you kindly review it for me

Reply:
> @topeogunleye Please brother I am done can you kindly review it for me

I can't. It could be anyone who will claim your review but I'll check if you've made the required changes I requested.

Reply:
@topeogunleye thanks alot, please I am waiting for you

Reply:
> @topeogunleye thanks alot, please I am waiting for you

@FredrickOtabil. You can go ahead and make a submission for review

Reply:
@topeogunleye thanks alot brother